Update specificatie.md

lvanderzande opened the pull request:
Tekstuele aanpassingen

lvanderzande committed:
Update specificatie.md

lvanderzande committed:
Update specificatie.md

Aanpassingen navragen sessie maandag 15-12

edwindoit commented on specificatie.md:
```diff
@@ -2,48 +2,55 @@
 - Informatie kan opgehaald worden bij het bronsysteem en getoond in het doelsysteem
- In de documentatie is vastgelegd welke informatie (ZIB's) er beschikbaar gemaakt kan worden via deze koppeling (Zie hoofdstuk ‘Lijst met ZIB’s’). Indien informatie opgehaald wordt, zal deze ook getoond worden in een vorm die passend is in het doelsysteem. Dit geldt ook voor gegevens uit bijvoorbeeld de Patient ZIB. Passend kan zijn als er bijvoorbeeld discrepatenties zijn deze tonen. Er is een l ...
+ In de documentatie is vastgelegd welke informatie (ZIB's) er beschikbaar gemaakt kan worden via de koppeling (Zie hoofdstuk ‘Lijst met ZIB’s’). Indien informatie opgehaald wordt, zal deze ook getoond worden in een vorm die passend is in het doelsysteem. Dit geldt ook voor gegevens uit bijvoorbeeld de Patient ZIB. Passend kan zijn als er bijvoorbeeld discrepatenties zijn deze tonen. Er is een lij ...
 - De medewerkers blijven in hun eigen systeem werken. De leveranciers zijn zelf verantwoordelijk hoe zij de medewerker het beste willen/kunnen ondersteunen.
```
Wat wordt hiermee bedoeld? Het wekt de indruk dat dedicated viewer-applicaties, zoals de HINQ ZNO-viewer, expliciet buiten scope vallen.

HugoFvanderGeest commented on specificatie.md:
```diff
@@ -178,11 +187,39 @@ POST /fhir/Patient/_search
 - Body: identifier=http://fhir.nl/fhir/NamingSystem/bsn|{bsn}
 
+
+**Bechikbare informatie voor de (wijk)verpleegkundinge in het Huiarts Dossier**
```
Er was volgens mij ook iets met dat er verschil in welke FHIR versie een HIS ondersteunde, dat kon dacht ik zijn R4 en STU3? Ik zou verwachten dat in deze documentatie ook beschreven staat het mechanisme van hoe we dat als ECD gaan achterhalen als we meerdere fhir versies toe staan, dat is de eerste stap in het proces voor als je inzage gaat uitvoeren. Daar zie ik hier nog niks van staan en had ik ook verwacht als stap te zien in de sequence diagram. Daarnaast krijg je wellicht dan ook 2 tabellen(of een extra colomn), want ik zou niet zeker weten of de URL-en hetzelfde blijven als je werkt met andere fhir versies werkt.

Loektimmermans replied:
Als ik mij goed herinner was gezegd dat een bronsysteem ofwel
STU3 en zib2017
of anders
R4 en zib2020

HugoFvanderGeest reviewed (changes requested):
comments

edwindoit reviewed (dismissed):
Bij deze twee inhoudelijke comments. Een aanvullende suggestie is om het geheel nog even door een LLM te halen om spel- en taalfouten te corrigeren.

hansvr2022 commented on specificatie.md:
```diff
@@ -68,10 +75,10 @@ Beide tools geven instructies om stappen 9-10 uit te kunnen voeren.
+De grondslag voor uitwisseling is de toestemming (consent) die vastgelegd is bij de patient. 
 
 ### Consent specificatie
 Consent: de verantwoordelijkheid voor het vastleggen en checken van consent wordt ingevuld door het bronsysteem zelf.
```
Dit hadden we inderdaad zo besproken.

Loektimmermans committed:
Update fhir resources table specificatie.md

Aanvulling van de tabel fhir resources en endpoints uit HIS opvraagbaar voor VVT

lvanderzande committed:
Merge pull request #46 from Loektimmermans/patch-1

Update fhir r4 resources table specificatie.md
